msgpack/fallback.py

```diff
@@ -696,7 +696,7 @@ def _pack(self, obj, nest_limit=DEFAULT_RECURSE_LIMIT,
                 else:
                     raise PackValueError("Bytes is too large")
                 return self._buffer.write(obj)
-            if check(obj, (Unicode, bytes, memoryview)):
+            if check(obj, (Unicode, bytes)):
```

Comment:
Having separated memoryview block is OK.
But such a large duplicated code block seems bad.
Please split code like `msgpack_pack_raw` and `msgpack_pack_bin` in C version.
Exception message can be build by `"%s object is too large" % (type(obj),)`